homeassistant/components/neato.py

```diff
@@ -126,3 +128,9 @@ def update_robots(self):
         _LOGGER.debug('Running HUB.update_robots %s',
                       self._hass.data[NEATO_ROBOTS])
         self._hass.data[NEATO_ROBOTS] = self.my_neato.robots
+        self._hass.data[NEATO_MAP_DATA] = self.my_neato.maps
+
+    @Throttle(timedelta(seconds=10))
```

Comment:
This is a very weird thing to throttle, since you pass in a destination path. It means that you might call something and the destination path will not work. It also means that if you have 2 robots, only 1 can show a map.

Why are you throttling ? You already only fetch a new map image if the url changes?